src/ZenstruckFoundryBundle.php

```diff
@@ -87,6 +89,16 @@ public function configure(DefinitionConfigurator $definition): void
                             ->example('my_instantiator')
                             ->defaultNull()
                         ->end()
+                        ->arrayNode('validation')
+                            ->info('Automatically validate the objects created.')
+                            ->canBeEnabled()
```

Comment:
it seems logical to add this config under `instantiator` conf, but this could be moved outisde from it.